track_local_static.go

```diff
@@ -382,6 +439,21 @@ func addExtensions(sample media.Sample, packets []*rtp.Packet) error {
 		}
 	}
 
+	if hyperscaleEncryption {
+		// now check wethear the rest of the packets need to be encrypted
+		for i := 1; i < len(packets); i++ {
+			sampleAttr = 0
+			// set the 'skip encryption' bit
+			if !encryption.ShouldEncrypt(sample, i, payloadDataIdx) {
```

Comment:
And I wonder.. is there a way to break this loop based on the encryption algorithm instead of iterating all the packets?

Reply:
never mind.. it's actually marking the SKIP encryption so it'll be required on all packets